### PR TITLE
Make Auth0 cacheLocation Local Storage

### PR DESCRIPTION
**Pull request by amy-corson-ibigroup**
**Description:**
Sets the auth0Provider cache location to localstorage. This prevents users from being signed out on page refresh.

**PR Checklist:**
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

File: example-config.yml [hunk 1]
```diff
@@ -68,6 +68,7 @@ persistence:
 #  enabled: true
 #  strategy: otp_middleware
 #  auth0:
+#    cacheLocation: localstorage (optional - default is "memory")
```
Comment (binh-dam-ibigroup) [inline example-config.yml:71]: Please add this new attribute to `Auth0Config` in config-types.ts.

**Review by binh-dam-ibigroup — APPROVED** [review]
It is working as intended on Chrome at least. We'll have to figure out something for Firefox and Safari. Please add the new config property to config-types before proceeding.

File: example-config.yml (hunk 1 above)
Comment (binh-dam-ibigroup) [reply example-config.yml:71]: Make the values a type that accepts two strings, i.e. `cacheLocation: 'memory' | 'localstorage'` or the equivalent type from the Auth0 library.

**Comment by binh-dam-ibigroup** [comment]
> We'll have to figure out something for Firefox and Safari.

Correction: That settings works (I testing using the wrong config file...).

**Review by miles-grant-ibigroup — APPROVED** [review]
What could go wrong!

**Pull request merged**